.github/workflows/mixin.yml

```diff
@@ -0,0 +1,32 @@
+name: mixin
+on:
+  push:
+    paths:
+    - 'docs/mixin/alerts/*.yml'
+  pull_request:
+    paths:
+    - 'docs/mixin/alerts/*.yml'
+jobs:
+  check-mixin:
+    runs-on: ubuntu-latest
+    name: Generate yaml
```

Comment:
```suggestion
    name: Test alert configuration syntax
```

Reply:
Copy-paste issue :facepalm: I changed it to something more generic as in the near future we will be expanding this with dashboard linting and potentially alert testing.